<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>644</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>8200</width>
    <height>6000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>deb-check</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_check">
     <attribute name="title">
      <string>检查包名</string>
     </attribute>
     <widget class="QTextEdit" name="text_name">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>90</y>
        <width>241</width>
        <height>471</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>551</width>
        <height>91</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="text_fullPackageName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>551</width>
        <height>331</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_copy">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>430</y>
        <width>106</width>
        <height>36</height>
       </rect>
      </property>
      <property name="text">
       <string>一键复制</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>60</y>
        <width>63</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>软件名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>651</width>
        <height>36</height>
       </rect>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>在此输入crp地址</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_check">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>10</y>
        <width>101</width>
        <height>36</height>
       </rect>
      </property>
      <property name="text">
       <string>check</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>63</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>包名</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>437</y>
        <width>63</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>安装命令</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_compare">
     <attribute name="title">
      <string>版本比较</string>
     </attribute>
     <widget class="QLabel" name="label_v2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>220</y>
        <width>63</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Version2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_msg">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>320</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_v1">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>170</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_v1">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>63</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Version1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_v2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>220</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_compare">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>270</y>
        <width>106</width>
        <height>36</height>
       </rect>
      </property>
      <property name="text">
       <string>compare</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
